Sources/Datadog/Core/Feature.swift

```diff
@@ -81,14 +81,12 @@ internal struct FeatureStorage {
         )
 
         let unauthorizedFileWriter = FileWriter(
-            dataFormat: dataFormat,
             orchestrator: unauthorizedFilesOrchestrator,
             encryption: commonDependencies.encryption,
             telemetry: telemetry
         )
```

Comment:
Great to see this change 👍. It means that writer stack is less dependant on the feature specifics and its ultimate decoupling is a key goal for V2.